scripts/normalize_user_tags.py

```diff
@@ -0,0 +1,187 @@
+from datetime import datetime
+import pytz
+import logging
+from django.apps import apps
+from website.app import init_app
+from scripts import utils as script_utils
+import sys
+from django.db import transaction
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+
+PROVIDER_SOURCE_TAGS = [
+    ('africarxiv_preprints', 'source:provider|preprint|africarxiv'),
+    ('agrixiv_preprints', 'source:provider|preprint|agrixiv'),
+    ('arabixiv_preprints', 'source:provider|preprint|arabixiv'),
+    ('bitss_preprints', 'source:provider|preprint|metaarxiv'),
```

Comment:
BITSS is now renamed to MetaArxiv.

Reply:
Are there no existing `metaarxiv_preprints` tags?